### ISSUE TITLE
Woocommerce integration requires to modify "item_code"

### ISSUE THREAD

**Issue by tecnofoodmx** [frappe/erpnext#21050]
In order to be able to integrate wocommerce with erpnext, the item code should have the following format: "wocommerce - " + "wocommerce_id".

This is highly inconvenient, specially when trying to integrate woocommerce to an existing erpnext company, since it requires either o of the following scenarios:
1- Modify an existing item code: theres plenty of room for operational disaster when trying to change item codes "on the fly". Completely unacceptable for a running business,
2- Create special items just to interact with wocommerce: much of the point of doing the integration is lost since it would require to manage special stock o to rework the generated sales order to match the actual item codes.

¿is it possible to fix this such that woocommerce interacts with ERPNext via the woocommerce_id (in a similar way as the shopify integration worked (by the way, Shopify worked beautifully)?

**Comment by jryandechavez** [comment]
I agree on this, currently we are also testing the integration of erpnext to woocommerce and this is one of our major concern. Erpnext created new item even if the SKU and description in woocommerce is the same as the item in Erpnext

**Comment by mikezareno** [comment]
There's a similar issue with Shopify integration. I think the fix is to allow alternate item lookup codes (woocommerce codes, shopify codes) that return the base Item Code. I'd be willing to chip in for development

**Comment by tecnofoodmx** [comment]
ANyone knows if it has been addressed?

**Comment by Thijsz0rz** [comment]
+1, this would help us as well!
